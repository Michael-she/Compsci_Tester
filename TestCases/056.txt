l a 3 3
d a 6 6
#
3 3 d
2 3 d
6 6 u
7 6 u
1 3 d
0 3 r
8 6 u
9 6 u
