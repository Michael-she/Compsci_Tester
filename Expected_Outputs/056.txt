pygame 2.5.2 (SDL 2.28.3, Python 3.8.10)
Hello from the pygame community. https://www.pygame.org/contribute.html
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  | A|  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
3 |  |  |  | a|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  | A|  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
3 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  | a|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  | A|  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
3 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  |  | a|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  | A|  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
3 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  |  | a|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  | A|  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
3 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  |  | a|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  | A|  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
3 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  |  | a|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  | A|  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
3 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  |  |  | a|  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  |  |  |  | A|  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
3 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  |  |  | a|  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
ERROR: Cannot move beyond the board
